### DISCUSSION TITLE
Make the Kestrel mission available earlier?

### DISCUSSION THREAD

**Discussion by brendanjones** [endless-sky/endless-sky#9980] [Balance, Mechanics, and Interactions]
By the time your combat rating is high enough to trigger the Kestrel missions, it’s probably not that useful to you. What do people think about lowering the combat rating required? 

Then the Kestrel can actually be a progression from other human warships like the Falcon and Leviathan, before the player moves on to alien ships. 

As someone said on the wiki https://endless-sky.fandom.com/wiki/Kestrel:

> I think that the combat rating requirement should be lowered to 4000 points for first playthroughs, and there should be a new recurring shipping or passenger mission that leads players to Tarazed. Ideally, these changes would allow new players to unlock the Kestrel in time for it to be a powerful flagship compared to the alternatives, and it would still make the Kestrel a partially hidden ship that takes skill and effort to access while also giving a breadcrumb mission to help new players find it.

(I don’t know if 4000 is the right number, for the record )

**Comment by Zitchas** [comment]
There was some discussion about this in the past, and a minor expansion of the Kestrel's availability for subsequent playthroughs: https://github.com/endless-sky/endless-sky/pull/8578

I'd probably be in favor of a variety of additional options for unlocking it faster; such as having completed any of the human campaigns or acquired the Emerald Sword. Perhaps dropping the initial "pure combat" rating from 8k down to 6k.

**Reply by brendanjones** [reply]
Ah thanks, I searched issues not PRs so missed that one. 

Weird - it looks like it's merged but in the [mission file](https://github.com/endless-sky/endless-sky/blob/be8965222ddfce8b05e454500125ba4409341d52/data/human/kestrel.txt) the combat rating required is still 8000? Was only the "combat rating" > 2000 has "global: unlocked kestrel" added and not the reduction from 8000 >> 6000?

I definitely agree with dropping the CR to 6000 for the initial play through, and having checked the combat ratings now I'd actually say drop it to 4000. Even a CR of 2000 is annoyingly late (on subsequent play-throughs). 

I mean, iirc you can start demanding tribute from planets at combat rating 7, which is 1097. I find a little incongruous the idea that you need about 6x the combat rating required to dominate entire planets in order to be considered a worthy enough pilot to test the Kestrel.

**Reply by Zitchas** [reply]
The PR description is erroneous. It never actually reduced the CR to 6000, but left it at 8000. The only actual change was adding in the "faster the second time" mechanic. 

It originally talked about doing so, though. I'd recommend reading the full conversation for more details.

**Reply by brendanjones** [reply]
So if I were to submit a PR dropping the combat rating to 6k, would it be considered/accepted? I don’t see anything in that linked PR explicitly rejecting the drop to 6k.

**Reply by Zitchas** [reply]
It might be. I would be in favor of it, but I am just one of many. It would get some discussion, in all likelihood.

**Reply by brendanjones** [reply]
Yeah no worries, I realise you don't speak for everyone. Simply checking if it'd be rejected outright :)

Change is here: https://github.com/endless-sky/endless-sky/pull/9988
